org.alienarena.alienarena.yaml

```diff
@@ -0,0 +1,35 @@
+id: org.alienarena.alienarena
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/src/alienarena/aa/source/bin/alienarena
+modules:
+  - name: alienarena
+    buildsystem: simple
+    no-autogen: true
+    sources:
+      - type: svn
+        url: svn://svn.icculus.org/alienarena/tags/7.71.4
+    build-commands:
+      - ./configure --prefix=/app/src/alienarena/aa/source
+      - make -j$(nproc)
+      - make install
+      - mkdir -p /app/share/icons/hicolor/256x256/apps/
+      - mkdir -p /app/share/applications
+
+      - echo -e "[Desktop Entry]\nName=Alien Arena\nComment=A standalone 3D first person online deathmatch shooter\nComment[es]=Un juego de acción en primera persona sobre red\nComment[it]=Sparatutto online 3D in prima persona\nComment[de]=Ein Internet- und netzwerkfähiger 3D Ego-Shooter\nExec=/app/src/alienarena/aa/source/bin/alienarena\nIcon=org.alienarena.alienarena\nTerminal=false\nType=Application\nCategories=Game;ArcadeGame;\nKeywords=aliens;fps;shooter;multiplayer;" > ./org.alienarena.alienarena.desktop
```

Comment:
this should be a file instead. But if there is a desktop file upstream then it should be used.

Reply:
Like this file. https://svn.icculus.org/alienarena/tags/7.71.4/unix_dist/alien-arena.desktop?view=log

Reply:
How would I use that one if the exec is /usr/games/alienarena

Reply:
but TBF if `/usr/games/alienarena` is hardcoded it's wrong because that doesn't even honour the prefix.